### PR TITLE
Created_works_section

### PR DESCRIPTION
**Pull request by ntzwilly**
### Build the middle section of the mobile website which include the<em> Work section</em> and the <em>About me</em> section:
- Use CSS Grid and Flexbox to layout the elements in the "Works" section. Use CSS Grid in the main container that contains all of the cards and use Flexbox to layout an individual card.

- No need to implement the detail modal window that requires JavaScript.

- Stick to the design as much as possible (e.g., font, colors, images, tests, margins) using the templates in Figma.

- Implement the button interactions (enable, hover, etc.).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,8 +34,142 @@
...
+      <button class="btn-language">html</button>
+      <button class="btn-language">css</button>
+      <button class="btn-language">javaScript</button>
+    </div>
+    <div>
+      <button class="btn-project">See Project</button>
+    </div>
+  </section>
+  <section class="grid-container">
+    <div class="image-item"><img src="images/portfolio4.png" alt="portfolio_snapshoot"></div>
```
Comment (DeliceLydia) [inline index.html:110]: - [ ] Here as well you need to use a background image not img tag

File: index.html [hunk 2]
```diff
@@ -34,8 +34,142 @@
       </ul>
     </section>
   </main>
-  
-  
+  <section class="grid-container">
+    <div class="image-item"><img src="images/portfolio1.png" alt="portfolio_snapshoot"></div>
+    <div class="title-item">Tonic</div>
```
Comment (DeliceLydia) [inline index.html:39]: - [ ] Project title is supposed to be a header please use it

File: index.html [hunk 3]
```diff
@@ -34,8 +34,142 @@
...
+      <button class="btn-language">css</button>
+      <button class="btn-language">javaScript</button>
+    </div>
+    <div>
+      <button class="btn-project">See Project</button>
+    </div>
+  </section>
+  <section class="grid-container">
+    <div class="image-item"><img src="images/portfolio3.png" alt="portfolio_snapshoot"></div>
+    <div class="title-item">Tonic</div>
```
Comment (DeliceLydia) [inline index.html:87]: - [ ] Please use a header tag

File: index.html [hunk 4]
```diff
@@ -34,8 +34,142 @@
...
+      <button class="btn-language">css</button>
+      <button class="btn-language">javaScript</button>
+    </div>
+    <div>
+      <button class="btn-project">See Project</button>
+    </div>
+  </section>
+  <section class="grid-container">
+    <div class="image-item"><img src="images/portfolio4.png" alt="portfolio_snapshoot"></div>
+    <div class="title-item">Tonic</div>
```
Comment (DeliceLydia) [inline index.html:111]: - [ ] Please use a header tag to all your project name

File: index.html [hunk 5]
```diff
@@ -34,8 +34,142 @@
...
+    </div>
+    <div class="text-item">
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    </div>
+    <div class="flex-button">
+      <button class="btn-language">html</button>
+      <button class="btn-language">css</button>
+      <button class="btn-language">javaScript</button>
+    </div>
+    <div>
```
Comment (DeliceLydia) [inline index.html:57]: - [ ] The list of stack used on the project need to be a ul tags and each one is a li kindly fiix that to all the projects

File: index.html [hunk 6]
```diff
@@ -34,8 +34,142 @@
...
+    </div>
+    <div class="text-item"><br>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p><br><br>
+    </div>
+    <div class="flex-button1">
+      <button class="btn-language">html</button>
+      <button class="btn-language">css</button>
+      <button class="btn-language">javaScript</button>
+    </div>
+    <div>
```
Comment (DeliceLydia) [inline index.html:81]: - [ ] Please use ul tags and li tag to each one of them

File: index.html [hunk 7]
```diff
@@ -34,8 +34,142 @@
...
+    </div>
+    <div class="text-item">
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    </div>
+    <div class="flex-button">
+      <button class="btn-language">html</button>
+      <button class="btn-language">css</button>
+      <button class="btn-language">javaScript</button>
+    </div>
+    <div>
```
Comment (DeliceLydia) [inline index.html:105]: - [ ] Here as well use ul and li tags

File: index.html [hunk 8]
```diff
@@ -34,8 +34,142 @@
...
```
Comment (DeliceLydia) [inline index.html:129]: - [ ] Kindly use ul and li tags

Comment (DeliceLydia) [inline index.html:129]: - [ ] Kindly use ul and li tags

File: index.html [hunk 9]
```diff
@@ -34,8 +34,142 @@
...
+        <p>Frameworks</p>
+        <i class="fas fa-chevron-right"></i>
+      </div>
+      <hr class="line">
+      <div class="language-div2">
+        <p>Skills</p>
+        <i class="fas fa-chevron-right"></i>
+      </div>
+      <hr class="line">
+    </section>
```
Comment (DeliceLydia) [inline index.html:171]: - [ ] Language, frameworks and skills are supposed to be in the ul tag and each one of them needs to have a li tag
- [ ] Technology in each category is supposed to be in the ul tag and use li tag to each one of them

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hello @ntzwilly ✌️, 

Good job so far! your project is looking nice already on the mobile, however, there are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ntzwilly** [comment]
Hi

Thank you for your review.

I have made the required changes. Kindly check again.

With thanks,
Willy

File: styles.css [hunk 10]
```diff
@@ -117,3 +118,303 @@ main {
...
+
+.item3,
+.item5 {
+  font-family: "Poppins", sans-serif;
+  font-size: 13px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 16px;
+  text-align: left;
+  margin-left: 10px;
```
Comment (Dipeshtwis) [inline styles.css:209]: [Optional]
- It's good that you used `item3` and `item5` class together but you can also use `.item1` class with `.item3` and `.item5` and other property in separate.

File: styles.css [hunk 11]
```diff
@@ -117,3 +118,303 @@ main {
...
+}
+
+.language-div p {
+  font-family: "Poppins", sans-serif;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 24px;
+  text-align: left;
+}
```
Comment (Dipeshtwis) [inline styles.css:333]: [OPTIONAL]
- [ ] `language-div p` and `language-div2 p` have same property, you are supposed to use together separating with comma. Like this

```
.language-div p,
.language-div2 p {
 ......
}
```

**Review by Dipeshtwis — APPROVED** [review]
# Hi @ntzwilly, 

### Status: Approved ✔️

:+1:  Professional repo, PR title, description, and README is looking awesome, keep up the good work :muscle:
:+1: The project setup is well organized. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Follow DRY ( Do not repeat yourself ) principle in CSS file. Check the comments under the review.


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**